admin/extensions.php

```diff
@@ -72,22 +72,25 @@
 								<li>
 									<?php if ( in_array( strtolower( $extension->title->rendered ), array_keys( $installed_plugins ) ) ) : ?>
 
-										<span class="button button-disabled" title="<?php esc_attr_e( 'This extension is already installed', 'backupwordpress' ); ?>"><?php _e( 'Installed', 'backupwordpress' ); ?></span>
+										<span class="button button-disabled" title="<?php esc_attr_e( 'This extension is already installed', 'backupwordpress' ); ?>"><?php esc_html_e( 'Installed', 'backupwordpress' ); ?></span>
 
 									<?php else : ?>
 
-										<a class="install-now button-primary" data-slug="<?php echo esc_attr( $extension->slug ); ?>" href="<?php echo esc_url( $extension->link ); ?>" aria-label="<?php echo esc_attr( sprintf(
-										__( 'Install %s now', 'backupwordpress' ),
+										<a class="install-now button-primary" data-slug="<?php echo esc_attr( $extension->slug ); ?>" href="<?php echo esc_url( $extension->link ); ?>" aria-label="<?php printf(
+										/* translators: Extension name */
+										esc_attr__( 'Install %s now', 'backupwordpress' ),
```

Comment:
This change actually removes escaping. `echo esc_attr()` needs to be used around the complete string, as `esc_attr__()` only escapes the translatable string, not the formatted string including `$extension->title->rendered`.